catroidTest/src/org/catrobat/catroid/uitest/ui/activity/PocketMusicTest.java

```diff
@@ -86,6 +89,40 @@ public void testNoteGridElement() {
 				.musicdroid_note_grid));
 	}
 
+	public void testAllButtonToggle() {
+		prepareTest();
+		solo.waitForActivity(PocketMusicActivity.class.getSimpleName());
+
+		TrackView trackView = (TrackView) solo.getCurrentActivity().findViewById(R.id.musicdroid_note_grid);
+		for (TrackRowView trackRowView : trackView.getTrackRowViews()) {
+			for (NoteView noteView : trackRowView.getNoteViews()) {
+				solo.clickOnView(noteView);
+				solo.sleep(200);
+				assertTrue("Button not toggled", noteView.isToggled());
+			}
+		}
+
+		for (TrackRowView trackRowView : trackView.getTrackRowViews()) {
+			for (NoteView noteView : trackRowView.getNoteViews()) {
+				solo.clickOnView(noteView);
+				solo.sleep(200);
+				assertTrue("Button not toggled", !noteView.isToggled());
```

Comment:
you could use `assertFalse("Button not toggled", noteView.isToggled())` here to make it easier to read :)